ecl/resource_ecl_provider_connectivity_tenant_connection_request_v2_test.go

```diff
@@ -0,0 +1,153 @@
+package ecl
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/nttcom/eclcloud"
+	"github.com/nttcom/eclcloud/ecl/provider_connectivity/v2/tenant_connection_requests"
+)
+
+func TestAccProviderConnectivityV2TenantConnectionRequest_basic(t *testing.T) {
+	var request tenant_connection_requests.TenantConnectionRequest
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheckTenantConnectionRequest(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckProviderConnectivityV2TenantConnectionRequestDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccProviderConnectivityV2TenantConnectionRequestBasic,
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
I think you should check if all attributes are set property here. Please look at `TestCheckResourceAttrSet`.

Reply:
confirm.